Remove remaining notions of streams (#123)
sframe-wg/sframe: draft-ietf-sframe-enc.md

HBH:
: Hop By Hop

We use "Selective Forwarding Unit (SFU)" and "media stream" in a less formal sense
than in {{?RFC7656}}.  An SFU is a selective switching function for media

> payloads, and a media stream a sequence of media payloads, in both cases

Reviewer: Should SFU and media stream be wrapped in quotations here?

Author: No, those are just to mark out the terms at introduction..

> payloads, and a media stream a sequence of media payloads, in both cases
> regardless of whether those media payloads are transported over RTP or some
> other protocol.
> 

Reviewer: If we wanted to omit saying RTP at all in this context:
```suggestion
payloads, and a media stream is a sequence of media payloads,
both agnostic of the transport protocol.
```

Author: This is the one instance where it's important to say RTP, since we're explicitly disclaiming it :)
